src/components/Col/_col.scss

```diff
@@ -0,0 +1,152 @@
+//-----------------------------
+// Grid.col
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/vars';
+@import '../../globals/scss/layout';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/helper-mixins';
+@import '@carbon/import-once/scss/import-once';
+@import '../../globals/scss/css--reset';
+
+@include exports('grid-col') {
+  .#{$prefix}--grid-col {
+    height: 100%;
+    box-sizing: border-box;
+    -webkit-box-flex: 0;
+  }
+  .#{$prefix}--col--span-2,
+  .#{$prefix}--col--lg--2 {
+    @include column-span(2) // flex: 0 0 (1/6 * 100%);
+      // max-width: (1/6 * 100%);
+;
+  }
+  .#{$prefix}--col--span-3,
+  .#{$prefix}--col--lg--3 {
+    @include column-span(3) // flex: 0 0 (1/4 * 100%);
+      // max-width: (1/4 * 100%);
+;
+  }
+  .#{$prefix}--col--span-4,
+  .#{$prefix}--col--lg--4 {
+    @include column-span(4);
+  }
+  .#{$prefix}--col--span-5,
+  .#{$prefix}--col--lg--5 {
+    @include column-span(5);
+  }
+  .#{$prefix}--col--span-6,
+  .#{$prefix}--col--lg--6 {
+    @include column-span(6);
+  }
+  .#{$prefix}--col--span-7,
+  .#{$prefix}--col--lg--7 {
+    @include column-span(7);
+  }
+  .#{$prefix}--col--span-8,
+  .#{$prefix}--col--lg--8 {
+    @include column-span(8);
+  }
+  .#{$prefix}--col--span-9,
```

Comment:
Looks good! One more thing you can do here is using a loop similar to this:

```
@for $i from 1 through 12 {
.#{$prefix}--col--span-#{$i}),
  .#{$prefix}--col--lg--#{$i}) {
    @include column-span(#{$i}));
  }
```

Reply:
sure @Utzel-Butzel